src/stores/TransactionStore.js

```diff
@@ -249,15 +264,20 @@ export default class TransactionStore {
   /**
    * Adds a reset approve tx to the queue.
    * @param {string} spender Address to reset the allowance for.
+   * @param {string} topicAddress Address of the TopicEvent. Only for Set Result and Vote.
+   * @param {string} oracleAddress Address of the Oracle. Only for Set Result and Vote.
    */
   @action
-  addResetApproveTx = async (spender) => {
+  addResetApproveTx = async (spender, topicAddress, oracleAddress) => {
     this.transactions.push(observable.object(new Transaction({
       type: TransactionType.RESET_APPROVE,
       senderAddress: this.app.wallet.currentAddress,
       receiverAddress: spender,
+      topicAddress,
+      oracleAddress,
```

Comment:
where are those two used? I don't get what do you mean by "track on detail page"

Reply:
Oh so we want to show reset histories to users